src/status_im2/contexts/chat/messages/view.cljs

```diff
@@ -59,13 +58,19 @@
   []
   (let [;;we want to react only on these fields, do not use full chat map here
         {:keys [chat-id show-input?] :as chat} (rf/sub [:chats/current-chat-chat-view])
-        mutual-contact-requests-enabled?       (rf/sub [:mutual-contact-requests/enabled?])]
+        mutual-contact-requests-enabled?       (rf/sub [:mutual-contact-requests/enabled?])
+        pinned-messages                        (rf/sub [:chats/pinned chat-id])
+        latest-pin-text                        (get-in (last (vals pinned-messages)) [:content :text])
+        pins-count                             (count (seq pinned-messages))]
     [rn/keyboard-avoiding-view {:style {:flex 1}}
      [page-nav]
      ;; TODO (flexsurfer) this should be in-app notification component in quo2
      ;; https://github.com/status-im/status-mobile/issues/14527
      [pin-limit-popover/pin-limit-popover chat-id]
-     [pin.banner/banner chat-id]
+     [quo/banner
+      {:latest-pin-text latest-pin-text
+       :pins-count      pins-count
+      }]
```

Comment:
You can merge the dangling brackets with the previous line.

Reply:
imo issues like this should be created as what we want from our auto formatter, I should not have to think about how stuff like this is presented. make lint-fix should just do this. 

Do we think this is possible? 
cc: @yqrashawn

Reply:
There's a balance @J-Son89 where the formatter causes more problems if it takes control of every aspect of the code style. We have that to be the case based on a PR opened a long time ago by @yqrashawn.

I know where you're coming from, something like Prettier, but formatting Lisp is not that clear cut and zprint has limitations at the moment. Lisps have more room for interpretation to be formatted, because an s-expression can appear anywhere, which is not the case in other programming languages that have the concept of statements vs expressions. Please, see my other comment about respecting newlines for more info.

It would be great if you or anyone else that wants to change the format rules, first change `.zprintrc` in your local machine, fully develop using the new rules and double-check if you really like what you see. If you do than by all means suggest a change!

Reply:
Also note that some of us are using zprint after every file save because it's really fast to format the current file, so there's no manual step to call `make lint-fix`. I would recommend you set this up if you haven't. I never use `make lint-fix`, first of all because it's painfully slow, but also because this would mean my editor is formatting using a different set of rules than what's specified in `.zprintrc`, which would be an error on my editor side.

So I use zprint just as I would use prettier in VS Code, no manual step to call `make lint-fix`.

Reply:
And on your point, I would love if zprint could respect new lines, but never leave dangling parentheses/brackets/etc

Reply:
cool, I'll accept that's it the best we can do for now and will involve some manual formatting. Thanks for the thorough explanation! will make sure my editor is setup correctly too

Reply:
> imo issues like this should be created as what we want from our auto formatter, I should not have to think about how stuff like this is presented. make lint-fix should just do this.
> 
> Do we think this is possible? cc: @yqrashawn

I've opened a feature request issue for zprint, currently it doesn't support merge dangling parenthesis if when respect-nl is enabled.

Reply:
Thank you @yqrashawn.

Reply:
> Also note that some of us are using zprint after every file save because it's really fast to format the current file, so there's no manual step to call `make lint-fix`. I would recommend you set this up if you haven't. I never use `make lint-fix`, first of all because it's painfully slow, but also because this would mean my editor is formatting using a different set of rules than what's specified in `.zprintrc`, which would be an error on my editor side.
> 
> So I use zprint just as I would use prettier in VS Code, no manual step to call `make lint-fix`.

for some reason `make lint-fix` does not seem to have effect as nothing really changes before and after. Am thinking is there a step i may have missed to make the command work with zprint?

Reply:
@jo-mut, could you share exactly the branch and file you're trying to format that's not working? I can try on my machine, just to confirm your suspicions.

Reply:
@ilmotta `make lint-fix` seems to be working now just fine now. I have no idea why wasnt before.

Reply:
Makes sense @jo-mut. Apparently there was a change in the command to collect files before they're passed in to zprint.

Reply:
nice thank you @ilmotta